commit 0e7a93944177473b2c7d125f75b632bf89022a25
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Aug 12 14:25:41 2021 -0300

    Modificações de particípio e de conj com POS diferentes

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index f3ddfce..77b8970 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -1618,7 +1618,7 @@
 25	lâmina	lâmina	NOUN	_	Gender=Fem|Number=Sing	22	obl	_	_
 26	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	28	cop	_	_
 27	1500	1500	NUM	_	NumType=Card	28	nummod	_	_
-28	m/s	m/s	NOUN	_	Gender=Masc|Number=Plur	9	conj	_	_
+28	m/s	m/s	NOUN	_	Gender=Masc|Number=Plur	9	parataxis	_	_
 29	,	,	PUNCT	_	_	32	punct	_	_
 30	após	após	ADP	_	_	32	case	_	_
 31	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	32	det	_	_
@@ -2024,7 +2024,7 @@
 10	atividades	atividade	NOUN	_	Gender=Fem|Number=Plur	9	nsubj:pass	_	_
 11	de	de	ADP	_	_	12	case	_	_
 12	campo	campo	NOUN	_	Gender=Masc|Number=Sing	10	nmod	_	_
-13	coordenadas	coordenada	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	10	acl	_	_
+13	coordenadas	coordenar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	10	acl	_	_
 14-15	pelo	_	_	_	_	_	_	_	_
 14	por	por	ADP	_	_	16	case	_	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	_	_
@@ -8421,9 +8421,9 @@
 28	retângulo	retângulo	NOUN	_	Gender=Masc|Number=Sing	25	obl	_	_
 29	vermelho	vermelho	ADJ	_	Gender=Masc|Number=Sing	28	amod	_	_
 30	,	,	PUNCT	_	_	31	punct	_	_
-31	falha	falha	NOUN	_	Gender=Fem|Number=Sing	24	parataxis	_	_
+31	falha	falha	NOUN	_	Gender=Fem|Number=Sing	33	nsubj	_	_
 32	rosa	roso	ADJ	_	Gender=Fem|Number=Sing	31	amod	_	_
-33	representa	representar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	31	conj	_	_
+33	representa	representar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	parataxis	_	_
 34	falha	falha	NOUN	_	Gender=Fem|Number=Sing	33	obj	_	_
 35	sintética	sintético	ADJ	_	Gender=Fem|Number=Sing	34	amod	_	_
 36	,	,	PUNCT	_	_	42	punct	_	_
@@ -8432,7 +8432,7 @@
 39	pretas	preto	ADJ	_	Gender=Fem|Number=Plur	38	amod	_	_
 40	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	42	cop	_	_
 41	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	42	det	_	_
-42	falhas	falha	NOUN	_	Gender=Fem|Number=Plur	31	conj	_	_
+42	falhas	falha	NOUN	_	Gender=Fem|Number=Plur	7	parataxis	_	_
 43	antitéticas	antitético	ADJ	_	Gender=Fem|Number=Plur	42	amod	_	_
 44	.	.	PUNCT	_	_	7	punct	_	_
 
@@ -11165,7 +11165,7 @@
 3	asfálticas	asfáltico	ADJ	_	Gender=Fem|Number=Plur	2	amod	_	_
 4	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	_	_
 5	ser	ser	AUX	_	VerbForm=Inf	6	aux:pass	_	_
-6	definidas	definido	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	4	xcomp	_	_
+6	definidas	definir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	4	xcomp	_	_
 7	como	como	ADP	_	_	8	case	_	_
 8	misturas	mistura	NOUN	_	Gender=Fem|Number=Plur	6	xcomp	_	_
 9	de	de	ADP	_	_	10	case	_	_
@@ -13421,21 +13421,21 @@
 28	;	;	PUNCT	_	_	5	punct	_	_
 29	não	não	ADV	_	Polarity=Neg	31	advmod	_	_
 30	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	31	expl	_	_
-31	decompõe	decompor	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	23	conj	_	_
+31	decompõe	decompor	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	conj	_	_
 32	,	,	PUNCT	_	_	31	punct	_	_
 33	deteriora	deteriorar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	31	conj	_	_
 34	ou	ou	CCONJ	_	_	35	cc	_	_
-35	apodrece	apodrecer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	33	conj	_	_
+35	apodrece	apodrecer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	31	conj	_	_
 36	;	;	PUNCT	_	_	38	punct	_	_
 37	não	não	ADV	_	Polarity=Neg	38	advmod	_	_
-38	atrai	atrair	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	33	conj	_	_
+38	atrai	atrair	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	conj	_	_
 39	cupins	cupim	NOUN	_	Gender=Masc|Number=Plur	38	obj	_	_
 40	ou	ou	CCONJ	_	_	41	cc	_	_
 41	insetos	inseto	NOUN	_	Gender=Masc|Number=Plur	39	conj	_	_
 42	;	;	PUNCT	_	_	45	punct	_	_
 43	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	45	aux:pass	_	_
 44	somente	somente	ADV	_	_	45	advmod	_	_
-45	atacado	atacar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	38	conj	_	_
+45	atacado	atacar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	5	conj	_	_
 46-47	pelo	_	_	_	_	_	_	_	_
 46	por	por	ADP	_	_	48	case	_	_
 47	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	48	det	_	_
@@ -13444,7 +13444,7 @@
 50	a	a	ADP	_	_	51	case	_	_
 51	quente	quente	NOUN	_	Gender=Masc|Number=Sing	48	nmod	_	_
 52	;	;	PUNCT	_	_	53	punct	_	_
-53	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	45	conj	_	_
+53	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	conj	_	_
 54	absorver	absorver	VERB	_	VerbForm=Inf	53	xcomp	_	_
 55	até	até	ADP	_	_	57	case	_	_
 56	cinco	cinco	NUM	_	NumType=Card	57	nummod	_	_
@@ -13456,9 +13456,9 @@
 62	água	água	NOUN	_	Gender=Fem|Number=Sing	60	nmod	_	_
 63	,	,	PUNCT	_	_	54	punct	_	_
 64	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	65	cop	_	_
-65	lubrificante	lubrificante	NOUN	_	Gender=Masc|Number=Sing	53	xcomp	_	_
+65	lubrificante	lubrificante	NOUN	_	Gender=Masc|Number=Sing	5	xcomp	_	_
 66	e	e	CCONJ	_	_	67	cc	_	_
-67	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	65	conj	_	_
+67	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	conj	_	_
 68	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	69	det	_	_
 69	características	característica	NOUN	_	Gender=Fem|Number=Plur	67	obj	_	_
 70	necessárias	necessário	ADJ	_	Gender=Fem|Number=Plur	69	amod	_	_
@@ -13476,7 +13476,7 @@
 3	tornam	tornar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	_	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	_	_
 5	vermiculita	vermiculita	NOUN	_	Gender=Fem|Number=Sing	3	obj	_	_
-6	expandida	expandido	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	5	acl	_	_
+6	expandida	expandir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	5	acl	_	_
 7	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	8	det	_	_
 8	produto	produto	NOUN	_	Gender=Masc|Number=Sing	6	obj	_	_
 9	de	de	ADP	_	_	11	case	_	_
@@ -22721,9 +22721,9 @@
 27	de	de	ADP	_	_	28	case	_	_
 28	invertebrados	invertebrado	NOUN	_	Gender=Masc|Number=Plur	26	nmod	_	_
 29	e	e	CCONJ	_	_	30	cc	_	_
-30	vertebrados	vertebrar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	28	conj	_	_
-31	marinhos	marinho	NOUN	_	Gender=Masc|Number=Plur	28	conj	_	_
-32	preservados	preservar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	31	acl	_	_
+30	vertebrados	vertebrado	NOUN	_	Gender=Masc|Number=Plur	28	conj	_	_
+31	marinhos	marinho	ADJ	_	Gender=Masc|Number=Plur	30	amod	_	_
+32	preservados	preservar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	30	acl	_	_
 33-34	na	_	_	_	_	_	_	_	_
 33	em	em	ADP	_	_	35	case	_	_
 34	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	35	det	_	_
@@ -23137,9 +23137,9 @@
 13	de	de	ADP	_	_	14	case	_	_
 14	moluscos	molusco	NOUN	_	Gender=Masc|Number=Plur	12	nmod	_	_
 15	,	,	PUNCT	_	_	16	punct	_	_
-16	dinoflagelados	dinoflagelar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	14	conj	_	_
+16	dinoflagelados	dinoflagelado	NOUN	_	Gender=Masc|Number=Plur	14	conj	_	_
 17	,	,	PUNCT	_	_	18	punct	_	_
-18	foraminíferos	foraminífero	ADJ	_	Gender=Masc|Number=Plur	14	conj	_	_
+18	foraminíferos	foraminífero	NOUN	_	Gender=Masc|Number=Plur	14	conj	_	_
 19	e	e	CCONJ	_	_	20	cc	_	_
 20	equinóides	equinóide	NOUN	_	Gender=Masc|Number=Plur	14	conj	_	_
 21	,	,	PUNCT	_	_	22	punct	_	_
@@ -23160,7 +23160,7 @@
 36	resíduos	resíduo	NOUN	_	Gender=Masc|Number=Plur	29	conj	_	_
 37	transgressivos	transgressivo	ADJ	_	Gender=Masc|Number=Plur	36	amod	_	_
 38	,	,	PUNCT	_	_	39	punct	_	_
-39	confirmando	confirmar	VERB	_	VerbForm=Ger	10	advcl	_	_
+39	confirmando	confirmar	VERB	_	VerbForm=Ger	27	advcl	_	_
 40	incursão	incursão	NOUN	_	Gender=Fem|Number=Sing	39	obj	_	_
 41	marinha	marinho	ADJ	_	Gender=Fem|Number=Sing	40	amod	_	_
 42	.	.	PUNCT	_	_	10	punct	_	_
@@ -26065,10 +26065,10 @@
 2	geometrias	geometria	NOUN	_	Gender=Fem|Number=Plur	5	nsubj:pass	_	_
 3	predominantes	predominante	ADJ	_	Gender=Fem|Number=Plur	2	amod	_	_
 4	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	5	aux:pass	_	_
-5	camadas	camada	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	_	_
+5	camadas	camada	NOUN	_	Gender=Fem|Number=Plur	0	root	_	_
 6	tabulares	tabular	ADJ	_	Gender=Fem|Number=Plur	5	amod	_	_
 7	centimétricas	centimétrico	ADJ	_	Gender=Fem|Number=Plur	5	amod	_	_
-8	podendo	poder	VERB	_	VerbForm=Ger	5	advcl	_	_
+8	podendo	poder	VERB	_	VerbForm=Ger	5	acl	_	_
 9	atingir	atingir	VERB	_	VerbForm=Inf	8	xcomp	_	_
 10	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	11	det	_	_
 11	metro	metro	NOUN	_	Gender=Masc|Number=Sing	9	obj	_	_
@@ -28359,7 +28359,7 @@
 15	TST	TST	PROPN	_	Gender=Masc|Number=Sing	13	obj	_	_
 16	(	(	PUNCT	_	_	17	punct	_	_
 17	empilhamento	empilhamento	NOUN	_	Gender=Masc|Number=Sing	15	appos	_	_
-18	retrogradacional	retrogradacional	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	17	acl	_	_
+18	retrogradacional	retrogradacional	ADJ	_	Gender=Masc|Number=Sing	17	amod	_	_
 19	)	)	PUNCT	_	_	17	punct	_	_
 20	(	(	PUNCT	_	_	21	punct	_	_
 21	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	15	appos	_	_
@@ -28539,7 +28539,7 @@
 8	sucede	suceder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	_	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	_	_
 10	empilhamento	empilhamento	NOUN	_	Gender=Masc|Number=Sing	8	obj	_	_
-11	retrogradacional	retrogradacional	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	10	acl	_	_
+11	retrogradacional	retrogradacional	ADJ	_	Gender=Masc|Number=Sing	10	amod	_	_
 12-13	do	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	14	case	_	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	_	_
@@ -30836,7 +30836,7 @@
 2	TST	TST	PROPN	_	Gender=Masc|Number=Sing	3	nsubj	_	_
 3	apresenta	apresentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	_	_
 4	empilhamento	empilhamento	NOUN	_	Gender=Masc|Number=Sing	3	obj	_	_
-5	retrogradacional	retrogradacional	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	4	acl	_	_
+5	retrogradacional	retrogradacional	ADJ	_	Gender=Masc|Number=Sing	4	amod	_	_
 6	,	,	PUNCT	_	_	5	punct	_	_
 7	com	com	ADP	_	_	9	case	_	_
 8	típico	típico	ADJ	_	Gender=Masc|Number=Sing	9	amod	_	_
@@ -32868,7 +32868,7 @@
 4	problema	problema	NOUN	_	Gender=Masc|Number=Sing	2	obj	_	_
 5	,	,	PUNCT	_	_	2	punct	_	_
 6	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	7	aux:pass	_	_
-7	investidas	investida	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	_	_
+7	investidas	investir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	_	_
 8	tecnologias	tecnologia	NOUN	_	Gender=Fem|Number=Plur	7	nsubj:pass	_	_
 9	de	de	ADP	_	_	10	case	_	_
 10	elevação	elevação	NOUN	_	Gender=Fem|Number=Sing	8	nmod	_	_
@@ -35385,14 +35385,14 @@
 4	e	e	CCONJ	_	_	5	cc	_	_
 5	magnética	magnético	ADJ	_	Gender=Fem|Number=Sing	3	conj	_	_
 6	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	7	aux:pass	_	_
-7	definidas	definido	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	_	_
+7	definidas	definir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	_	_
 8-9	pelas	_	_	_	_	_	_	_	_
 8	por	por	ADP	_	_	10	case	_	_
 9	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	10	det	_	_
 10	Equações	Equações	PROPN	_	Gender=Fem|Number=Plur	7	obl:agent	_	_
 11	14	14	NUM	_	NumType=Card	10	nummod	_	_
 12	e	e	CCONJ	_	_	13	cc	_	_
-13	15	15	NUM	_	NumType=Card	10	conj	_	_
+13	15	15	NUM	_	NumType=Card	11	conj	_	_
 14	,	,	PUNCT	_	_	15	punct	_	_
 15	respectivamente	respectivamente	ADV	_	_	7	advmod	_	_
 16	.	.	PUNCT	_	_	7	punct	_	_
@@ -41175,7 +41175,7 @@
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	_	_
 5	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	2	nmod	_	_
 6	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	aux:pass	_	_
-7	classificado	classificado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
+7	classificado	classificar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
 8	como	como	ADP	_	_	10	case	_	_
 9	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	10	det	_	_
 10	método	método	NOUN	_	Gender=Masc|Number=Sing	7	xcomp	_	_
@@ -43375,9 +43375,9 @@
 16	forças	força	NOUN	_	Gender=Fem|Number=Plur	10	conj	_	_
 17	géis	gel	NOUN	_	Gender=Masc|Number=Plur	16	nmod	_	_
 18	e	e	CCONJ	_	_	19	cc	_	_
-19	filtrado	filtrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	17	conj	_	_
+19	filtrado	filtrado	NOUN	_	Gender=Masc|Number=Sing	17	conj	_	_
 20	em	em	ADP	_	_	21	case	_	_
-21	valores	valor	NOUN	_	Gender=Masc|Number=Plur	19	obl	_	_
+21	valores	valor	NOUN	_	Gender=Masc|Number=Plur	8	obl	_	_
 22	adequados	adequado	ADJ	_	Gender=Masc|Number=Plur	21	amod	_	_
 23	para	para	SCONJ	_	_	24	mark	_	_
 24	conferir	conferir	VERB	_	VerbForm=Inf	8	advcl	_	_
@@ -43399,7 +43399,7 @@
 38	boa	bom	ADJ	_	Gender=Fem|Number=Sing	39	amod	_	_
 39	taxa	taxa	NOUN	_	Gender=Fem|Number=Sing	35	obl	_	_
 40	e	e	CCONJ	_	_	41	cc	_	_
-41	capacidade	capacidade	NOUN	_	Gender=Fem|Number=Sing	39	conj	_	_
+41	capacidade	capacidade	NOUN	_	Gender=Fem|Number=Sing	29	conj	_	_
 42	de	de	ADP	_	_	43	case	_	_
 43	estabilização	estabilização	NOUN	_	Gender=Fem|Number=Sing	41	nmod	_	_
 44-45	das	_	_	_	_	_	_	_	_
@@ -44122,7 +44122,7 @@
 16	(	(	PUNCT	_	_	17	punct	_	_
 17	cascalhos	cascalho	NOUN	_	Gender=Masc|Number=Plur	15	appos	_	_
 18	)	)	PUNCT	_	_	17	punct	_	_
-19	agregados	agregado	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	15	acl	_	_
+19	agregados	agregar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	15	acl	_	_
 20	.	.	PUNCT	_	_	10	punct	_	_
 
 # text = O fluido é então imediatamente direcionado a um sistema de controle de sólidos (Figura II.3).
@@ -44941,12 +44941,12 @@
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	_	_
 26	forma	forma	NOUN	_	Gender=Fem|Number=Sing	28	nsubj:pass	_	_
 27	original	original	ADJ	_	Gender=Fem|Number=Sing	26	amod	_	_
-28	restabelecida	restabelecer	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	3	advcl	_	_
+28	restabelecida	restabelecer	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	14	advcl	_	_
 29	quando	quando	SCONJ	_	_	33	mark	_	_
 30	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	31	det	_	_
 31	tensão	tensão	NOUN	_	Gender=Fem|Number=Sing	33	nsubj:pass	_	_
 32	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	33	aux:pass	_	_
-33	retirada	retirada	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	28	advcl	_	_
+33	retirada	retirar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	28	advcl	_	_
 34	.	.	PUNCT	_	_	3	punct	_	_
 
 # text = Análises químicas mostram que as argilas são constituídas essencialmente por silício, alumínio e água, e, freqüentemente, por quantidades apreciáveis de ferro e de metais alcalinos e alcalinoterrosos (MACHADO, 1986).
@@ -46378,7 +46378,7 @@
 26	folhas	folha	NOUN	_	Gender=Fem|Number=Plur	29	nsubj:pass	_	_
 27	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	29	aux:pass	_	_
 28	usualmente	usualmente	ADV	_	_	29	advmod	_	_
-29	chamadas	chamado	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	5	conj	_	_
+29	chamadas	chamar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	5	conj	_	_
 30	tetraédricas	tetraédrico	ADJ	_	Gender=Fem|Number=Plur	29	xcomp	_	_
 31	e	e	CCONJ	_	_	32	cc	_	_
 32	folhas	folha	NOUN	_	Gender=Fem|Number=Plur	30	conj	_	_
@@ -47010,7 +47010,7 @@
 3	entre	entre	ADP	_	_	4	case	_	_
 4	camadas	camada	NOUN	_	Gender=Fem|Number=Plur	2	nmod	_	_
 5	ou	ou	CCONJ	_	_	6	cc	_	_
-6	interlamelar	interlamelar	VERB	_	VerbForm=Inf	4	conj	_	_
+6	interlamelar	interlamelar	ADJ	_	Gender=Masc|Number=Sing	4	conj	_	_
 7	varia	variar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	_	_
 8	com	com	ADP	_	_	10	case	_	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	_	_
@@ -48389,7 +48389,7 @@
 39	de	de	ADP	_	_	41	case	_	_
 40	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	41	det	_	_
 41	caulinita	caulinita	NOUN	_	Gender=Fem|Number=Sing	38	nmod	_	_
-42	aumenta	aumentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	29	conj	_	_
+42	aumenta	aumentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	29	parataxis	_	_
 43	também	também	ADV	_	_	42	advmod	_	_
 44	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	45	det	_	_
 45	capacidade	capacidade	NOUN	_	Gender=Fem|Number=Sing	42	obj	_	_
@@ -48419,7 +48419,7 @@
 67	devida	dever	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	12	conj	_	_
 68	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	69	det	_	_
 69	ligação	ligação	NOUN	_	Gender=Fem|Number=Sing	67	obj	_	_
-70	partida	partida	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	69	acl	_	_
+70	partida	partir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	69	acl	_	_
 71	e	e	CCONJ	_	_	74	cc	_	_
 72-73	à	_	_	_	_	_	_	_	_
 72	a	a	ADP	_	_	74	case	_	_
@@ -48451,7 +48451,7 @@
 94	;	;	PUNCT	_	_	97	punct	_	_
 95	essas	esse	DET	_	Gender=Fem|Number=Plur|PronType=Dem	96	det	_	_
 96	trocas	troca	NOUN	_	Gender=Fem|Number=Plur	97	nsubj	_	_
-97	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	74	conj	_	_
+97	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	67	parataxis	_	_
 98	ocorrer	ocorrer	VERB	_	VerbForm=Inf	97	xcomp	_	_
 99-100	na	_	_	_	_	_	_	_	_
 99	em	em	ADP	_	_	101	case	_	_
@@ -48464,7 +48464,7 @@
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	_	_
 2	precisão	precisão	NOUN	_	Gender=Fem|Number=Sing	16	nsubj	_	_
 3	com	com	SCONJ	_	_	11	mark	_	_
-4	que	que	PRON	_	_	13	obl	_	_
+4	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	13	obl	_	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	_	_
 6	capacidade	capacidade	NOUN	_	Gender=Fem|Number=Sing	11	nsubj	_	_
 7	de	de	ADP	_	_	8	case	_	_
@@ -48507,7 +48507,7 @@
 44	capacidade	capacidade	NOUN	_	Gender=Fem|Number=Sing	42	obj	_	_
 45	de	de	ADP	_	_	46	case	_	_
 46	troca	troca	NOUN	_	Gender=Fem|Number=Sing	44	nmod	_	_
-47	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	36	conj	_	_
+47	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	36	parataxis	_	_
 48	reagir	reagir	VERB	_	VerbForm=Inf	47	xcomp	_	_
 49	com	com	ADP	_	_	51	case	_	_
 50	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	51	det	_	_
@@ -50228,7 +50228,7 @@
 2	amostra	amostra	NOUN	_	Gender=Fem|Number=Sing	7	nsubj:pass	_	_
 3	seca	seca	ADJ	_	Gender=Fem|Number=Sing	2	amod	_	_
 4	e	e	CCONJ	_	_	5	cc	_	_
-5	pulverizada	pulverizado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	3	conj	_	_
+5	pulverizada	pulverizar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	3	conj	_	_
 6	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	7	aux:pass	_	_
 7	fundida	fundir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
 8	com	com	ADP	_	_	9	case	_	_
@@ -50240,7 +50240,7 @@
 14-15	à	_	_	_	_	_	_	_	_
 14	a	a	ADP	_	_	16	case	_	_
 15	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	16	det	_	_
-16	1000ºC	1000ºC	SYM	_	Gender=Masc|Number=Plur	7	obl	_	_
+16	1000ºC	1000ºC	SYM	_	_	7	obl	_	_
 17	por	por	ADP	_	_	19	case	_	_
 18	30	30	NUM	_	NumType=Card	19	nummod	_	_
 19	minutos	minuto	NOUN	_	Gender=Masc|Number=Plur	7	obl	_	_
@@ -50250,7 +50250,7 @@
 23	pastilha	pastilha	NOUN	_	Gender=Fem|Number=Sing	21	obj	_	_
 24	,	,	PUNCT	_	_	21	punct	_	_
 25	sobre	sobre	ADP	_	_	27	case	_	_
-26	a	o	DET	_	Gender=Fem|Number=Sing|PronType=Art	27	det	_	_
+26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	_	_
 27	qual	qual	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	28	obl	_	_
 28	incidiu	incidir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	23	acl:relcl	_	_
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	_	_
@@ -50386,7 +50386,7 @@
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	_	_
 2	amostra	amostra	NOUN	_	Gender=Fem|Number=Sing	4	nsubj:pass	_	_
 3	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	4	aux:pass	_	_
-4	pulverizada	pulverizado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
+4	pulverizada	pulverizar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
 5	e	e	CCONJ	_	_	6	cc	_	_
 6	prensada	prensar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	4	conj	_	_
 7	sobre	sobre	ADP	_	_	8	case	_	_
@@ -51219,7 +51219,7 @@
 10	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	11	aux:pass	_	_
 11	resfriadas	resfriar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	_	_
 12	e	e	CCONJ	_	_	13	cc	_	_
-13	abertas	aberto	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	11	conj	_	_
+13	abertas	abrir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	11	conj	_	_
 14	.	.	PUNCT	_	_	11	punct	_	_
 
 # text = Seu conteúdo foi cuidadosamente lavado em uma peneira 30 (abertura 0,12mm), o material foi seco em estufa a 80ºC durante 16h, o material foi peneirado usando-se as malhas 8 e 30 (4,76mm e 0,12mm) e finalmente pesado.
@@ -63453,14 +63453,14 @@
 9	G	g	NOUN	_	Gender=Masc|Number=Plur	8	nmod	_	_
 10	)	)	PUNCT	_	_	8	punct	_	_
 11	,	,	PUNCT	_	_	12	punct	_	_
-12	compostos	compor	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	5	conj	_	_
+12	compostos	compor	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	5	acl	_	_
 13	por	por	ADP	_	_	14	case	_	_
 14	clastos	clasto	NOUN	_	Gender=Masc|Number=Plur	12	obl:agent	_	_
 15	de	de	ADP	_	_	17	case	_	_
 16	diferentes	diferente	DET	_	Gender=Fem|Number=Plur|PronType=Ind	17	det	_	_
 17	dimensões	dimensão	NOUN	_	Gender=Fem|Number=Plur	14	nmod	_	_
 18	,	,	PUNCT	_	_	19	punct	_	_
-19	variando	variar	VERB	_	VerbForm=Ger	12	advcl	_	_
+19	variando	variar	VERB	_	VerbForm=Ger	14	acl	_	_
 20	de	de	ADP	_	_	22	case	_	_
 21	alguns	algum	DET	_	Gender=Masc|Number=Plur|PronType=Ind	22	det	_	_
 22	milímetros	milímetro	NOUN	_	Gender=Masc|Number=Plur	19	obl	_	_
@@ -63470,11 +63470,11 @@
 26	,	,	PUNCT	_	_	29	punct	_	_
 27	normalmente	normalmente	ADV	_	_	29	advmod	_	_
 28	bem	bem	ADV	_	_	29	advmod	_	_
-29	arredondados	arredondado	ADJ	_	Gender=Masc|Number=Plur	25	amod	_	_
+29	arredondados	arredondado	ADJ	_	Gender=Masc|Number=Plur	14	amod	_	_
 30	,	,	PUNCT	_	_	33	punct	_	_
 31	e	e	CCONJ	_	_	33	cc	_	_
 32	com	com	ADP	_	_	33	case	_	_
-33	matriz	matriz	NOUN	_	Gender=Fem|Number=Sing	25	conj	_	_
+33	matriz	matriz	NOUN	_	Gender=Fem|Number=Sing	29	conj	_	_
 34	arenosa	arenoso	ADJ	_	Gender=Fem|Number=Sing	33	amod	_	_
 35	.	.	PUNCT	_	_	3	punct	_	_
 
@@ -73741,7 +73741,7 @@
 # sent_id = 161-20150810-MONOGRAFIA_0-88
 1	Depois	depois	ADV	_	_	3	mark	_	MWEPOS=SCONJ
 2	de	de	ADP	_	_	1	fixed	_	_
-3	tratado	tratado	NOUN	_	Gender=Masc|Number=Sing	9	advcl	_	_
+3	tratado	tratar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	9	advcl	_	_
 4	e	e	CCONJ	_	_	5	cc	_	_
 5	resfriado	resfriar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	3	conj	_	_
 6	,	,	PUNCT	_	_	1	punct	_	_
@@ -78813,7 +78813,7 @@
 56	alto	alto	ADJ	_	Gender=Masc|Number=Sing	57	amod	_	_
 57	forno	forno	NOUN	_	Gender=Masc|Number=Sing	52	nmod	_	_
 58	e	e	CCONJ	_	_	59	cc	_	_
-59	segregadas	segregar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	57	conj	_	_
+59	segregadas	segregar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	49	conj	_	_
 60-61	do	_	_	_	_	_	_	_	_
 60	de	de	ADP	_	_	62	case	_	_
 61	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	62	det	_	_
@@ -78997,7 +78997,7 @@
 19	por	por	ADP	_	_	20	case	_	_
 20	laminadores	laminador	NOUN	_	Gender=Masc|Number=Plur	17	obl:agent	_	_
 21	e	e	CCONJ	_	_	22	cc	_	_
-22	transformado	transformar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	20	conj	_	_
+22	transformado	transformar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	17	conj	_	_
 23	em	em	ADP	_	_	24	case	_	_
 24	produtos	produto	NOUN	_	Gender=Masc|Number=Plur	22	obl:arg	_	_
 25	siderúrgicos	siderúrgico	ADJ	_	Gender=Masc|Number=Plur	24	amod	_	_
@@ -79011,7 +79011,7 @@
 32	transformação	transformação	NOUN	_	Gender=Fem|Number=Sing	30	nmod	_	_
 33	,	,	PUNCT	_	_	35	punct	_	_
 34	como	como	ADP	_	_	35	case	_	_
-35	chapas	chapa	NOUN	_	Gender=Fem|Number=Plur	30	nmod	_	_
+35	chapas	chapa	NOUN	_	Gender=Fem|Number=Plur	24	nmod	_	_
 36	grossas	grosso	ADJ	_	Gender=Fem|Number=Plur	35	amod	_	_
 37	e	e	CCONJ	_	_	38	cc	_	_
 38	finas	fino	ADJ	_	Gender=Fem|Number=Plur	36	conj	_	_
@@ -82261,7 +82261,7 @@
 
 # text = gismaps.com.br, permitindo que seja colocado no mapa base e posteriormente editado.
 # sent_id = 161-20150810-MONOGRAFIA_0-396
-1	gismaps	gismap	NOUN	_	Gender=Masc|Number=Sing	0	root	_	_
+1	gismaps	gismap	NOUN	_	Gender=Masc|Number=Sing	0	root	_	sentenciação
 2	.	.	PUNCT	_	_	3	punct	_	_
 3	com.br	com.br	NOUN	_	Gender=Masc|Number=Sing	1	appos	_	_
 4	,	,	PUNCT	_	_	5	punct	_	_
@@ -89459,7 +89459,7 @@
 18	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	20	cop	_	_
 19	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	20	det	_	_
 20	valor	valor	NOUN	_	Gender=Masc|Number=Sing	0	root	_	_
-21	estimado	estimado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	20	acl	_	_
+21	estimado	estimar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	20	acl	_	_
 22	baseado	basear	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	20	acl	_	_
 23-24	no	_	_	_	_	_	_	_	_
 23	em	em	ADP	_	_	25	case	_	_
@@ -100272,7 +100272,7 @@
 13	gás	gás	NOUN	_	Gender=Masc|Number=Sing	11	nmod	_	_
 14	natural	natural	ADJ	_	Gender=Masc|Number=Sing	13	amod	_	_
 15	e	e	CCONJ	_	_	16	cc	_	_
-16	condensado	condensado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	14	conj	_	_
+16	condensado	condensado	ADJ	_	Gender=Masc|Number=Sing	14	conj	_	_
 17	,	,	PUNCT	_	_	18	punct	_	_
 18	denominada	denominar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	11	acl	_	_
 19	Júpiter	Júpiter	PROPN	_	Gender=Fem|Number=Sing	18	obj	_	_
@@ -101055,7 +101055,7 @@
 4	que	que	SCONJ	_	_	8	mark	_	_
 5	tal	tal	DET	_	Gender=Masc|Number=Sing|PronType=Ind	6	det	_	_
 6	volume	volume	NOUN	_	Gender=Masc|Number=Sing	8	nsubj	_	_
-7	estimado	estimado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	6	acl	_	_
+7	estimado	estimar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	6	acl	_	_
 8	considera	considerar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	3	ccomp	_	_
 9	apenas	apenas	ADV	_	_	10	advmod	_	_
 10	volumes	volume	NOUN	_	Gender=Masc|Number=Plur	8	obj	_	_
@@ -101306,7 +101306,7 @@
 36	bopd	bopd	NOUN	_	Gender=Masc|Number=Plur	32	nmod	_	_
 37	)	)	PUNCT	_	_	32	punct	_	_
 38	e	e	CCONJ	_	_	39	cc	_	_
-39	inferir	inferir	VERB	_	VerbForm=Inf	15	conj	_	_
+39	inferir	inferir	VERB	_	VerbForm=Inf	3	conj	_	_
 40	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	41	det	_	_
 41	capacidades	capacidade	NOUN	_	Gender=Fem|Number=Plur	39	obj	_	_
 42-43	das	_	_	_	_	_	_	_	_
@@ -104198,7 +104198,7 @@
 25	(	(	PUNCT	_	_	27	punct	_	_
 26	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	27	det	_	_
 27	área	área	NOUN	_	Gender=Fem|Number=Sing	23	appos	_	_
-28	classificada	classificado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	27	acl	_	_
+28	classificada	classificar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	27	acl	_	_
 29	,	,	PUNCT	_	_	31	punct	_	_
 30	em	em	ADP	_	_	31	case	_	_
 31	termos	termos	NOUN	_	Gender=Masc|Number=Plur	27	nmod	_	_
@@ -110080,13 +110080,13 @@
 17	de	de	ADP	_	_	18	case	_	_
 18	fluxo	fluxo	NOUN	_	Gender=Masc|Number=Sing	16	nmod	_	_
 19	,	,	PUNCT	_	_	20	punct	_	_
-20	uma	um	NUM	_	NumType=Card	16	appos	_	_
+20	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	16	appos	_	_
 21	para	para	ADP	_	_	23	case	_	_
 22	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	23	det	_	_
 23	líquido	líquido	NOUN	_	Gender=Masc|Number=Sing	20	nmod	_	_
 24	,	,	PUNCT	_	_	20	punct	_	_
 25	onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	26	obl	_	_
-26	haverá	haver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	16	acl:relcl	_	_
+26	haverá	haver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	20	acl:relcl	_	_
 27	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	28	det	_	_
 28	bomba	bomba	NOUN	_	Gender=Fem|Number=Sing	26	obj	_	_
 29	monofásica	monofásico	ADJ	_	Gender=Fem|Number=Sing	28	amod	_	_
@@ -110096,11 +110096,11 @@
 32	tipo	tipo	NOUN	_	Gender=Masc|Number=Sing	28	nmod	_	_
 33	BCS	BCS	PROPN	_	Gender=Masc|Number=Sing	32	nmod	_	_
 34	para	para	SCONJ	_	_	35	mark	_	_
-35	suplementar	suplementar	NOUN	_	Gender=Masc|Number=Sing	26	obl	_	_
+35	suplementar	suplementar	VERB	_	VerbForm=Inf	26	advcl	_	_
 36	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	37	det	_	_
 37	energia	energia	NOUN	_	Gender=Fem|Number=Sing	35	nmod	_	_
 38	e	e	CCONJ	_	_	39	cc	_	_
-39	elevar	elevar	VERB	_	VerbForm=Inf	37	conj	_	_
+39	elevar	elevar	VERB	_	VerbForm=Inf	35	conj	_	_
 40	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	det	_	_
 41	líquido	líquido	NOUN	_	Gender=Masc|Number=Sing	39	obj	_	_
 42	até	até	ADP	_	_	44	case	_	_
@@ -110109,13 +110109,13 @@
 45	,	,	PUNCT	_	_	48	punct	_	_
 46	e	e	CCONJ	_	_	48	cc	_	_
 47	outra	outro	DET	_	Gender=Fem|Number=Sing|PronType=Ind	48	det	_	_
-48	linha	linha	NOUN	_	Gender=Fem|Number=Sing	28	conj	_	_
+48	linha	linha	NOUN	_	Gender=Fem|Number=Sing	20	conj	_	_
 49	para	para	ADP	_	_	51	case	_	_
 50	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	51	det	_	_
 51	gás	gás	NOUN	_	Gender=Masc|Number=Sing	48	nmod	_	_
 52	,	,	PUNCT	_	_	48	punct	_	_
 53	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	54	nsubj	_	_
-54	terá	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	51	acl:relcl	_	_
+54	terá	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	48	acl:relcl	_	_
 55	seu	seu	DET	_	Gender=Masc|Number=Sing|PronType=Prs	56	det	_	_
 56	fluxo	fluxo	NOUN	_	Gender=Masc|Number=Sing	54	obj	_	_
 57	ascendente	ascendente	ADJ	_	Gender=Masc|Number=Sing	56	amod	_	_
@@ -117102,7 +117102,7 @@
 7	água	água	NOUN	_	Gender=Fem|Number=Sing	5	conj	_	_
 8	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	_	_
 9	ser	ser	AUX	_	VerbForm=Inf	10	aux:pass	_	_
-10	classificada	classificado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	8	xcomp	_	_
+10	classificada	classificar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	8	xcomp	_	_
 11	como	como	ADP	_	_	12	case	_	_
 12	miscível	miscível	ADJ	_	Gender=Masc|Number=Sing	10	xcomp	_	_
 13	e	e	CCONJ	_	_	14	cc	_	_
@@ -117148,7 +117148,7 @@
 30	caso	caso	SCONJ	_	_	33	mark	_	_
 31	contrário	contrário	NOUN	_	Gender=Masc|Number=Sing	33	nsubj:pass	_	_
 32	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	33	aux:pass	_	_
-33	classificada	classificado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	27	advcl	_	_
+33	classificada	classificar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	27	advcl	_	_
 34	como	como	ADP	_	_	35	case	_	_
 35	imiscível	imiscível	NOUN	_	Gender=Masc|Number=Sing	33	xcomp	_	_
 36	.	.	PUNCT	_	_	27	punct	_	_
@@ -118539,10 +118539,10 @@
 21	não	não	ADV	_	Polarity=Neg	22	advmod	_	_
 22	tratado	tratar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	20	acl	_	_
 23	,	,	PUNCT	_	_	24	punct	_	_
-24	i.e.	i.e.	NOUN	_	Gender=Masc|Number=Sing	20	appos	_	_
+24	i.e.	i.e.	CCONJ	_	_	27	cc	_	_
 25	,	,	PUNCT	_	_	27	punct	_	_
 26	com	com	ADP	_	_	27	case	_	_
-27	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	20	nmod	_	_
+27	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	22	conj	_	_
 28	e	e	CCONJ	_	_	30	cc	_	_
 29	não	não	ADV	_	Polarity=Neg	30	advmod	_	_
 30	desidratado	desidratar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	27	conj	_	_
@@ -124374,7 +124374,7 @@
 5	)	)	PUNCT	_	_	4	punct	_	_
 6	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	9	cop	_	_
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	9	det	_	_
-8	chamadas	chamado	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	9	acl	_	_
+8	chamadas	chamado	ADJ	_	Gender=Fem|Number=Plur	9	amod	_	_
 9	emulsões	emulsão	NOUN	_	Gender=Fem|Number=Plur	0	root	_	_
 10	inversas	inverso	ADJ	_	Gender=Fem|Number=Plur	9	amod	_	_
 11	,	,	PUNCT	_	_	9	punct	_	_
@@ -128626,7 +128626,7 @@
 17	ocasionem	ocasionar	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	15	acl:relcl	_	_
 18	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	19	det	_	_
 19	parada	parada	NOUN	_	Gender=Fem|Number=Sing	17	obj	_	_
-20	inesperada	inesperado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	19	acl	_	_
+20	inesperada	inesperado	ADJ	_	Gender=Fem|Number=Sing	19	amod	_	_
 21-22	das	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	23	case	_	_
 22	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	23	det	_	_
@@ -128796,7 +128796,7 @@
 36	sofre	sofrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	24	acl:relcl	_	_
 37	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	38	det	_	_
 38	parada	parada	NOUN	_	Gender=Fem|Number=Sing	36	obj	_	_
-39	inesperada	inesperado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	38	acl	_	_
+39	inesperada	inesperado	ADJ	_	Gender=Fem|Number=Sing	38	amod	_	_
 40-41	das	_	_	_	_	_	_	_	_
 40	de	de	ADP	_	_	42	case	_	_
 41	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	42	det	_	_
@@ -130398,7 +130398,7 @@
 13	em	em	ADP	_	_	14	case	_	_
 14	produtos	produto	NOUN	_	Gender=Masc|Number=Plur	12	obl:arg	_	_
 15	petrolíferos	petrolífero	ADJ	_	Gender=Masc|Number=Plur	14	amod	_	_
-16	refinados	refinado	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	14	acl	_	_
+16	refinados	refinado	ADJ	_	Gender=Masc|Number=Plur	14	amod	_	_
 17	.	.	PUNCT	_	_	12	punct	_	_
 
 # text = Outro uso do óleo pesado é como combustível para estações de energia.
@@ -136167,7 +136167,7 @@
 11	e	e	CCONJ	_	_	12	cc	_	_
 12	21	21	NUM	_	NumType=Card	8	conj	_	_
 13	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	14	aux:pass	_	_
-14	abertas	aberto	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	1	parataxis	_	_
+14	abertas	abrir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	1	parataxis	_	_
 15	e	e	CCONJ	_	_	21	cc	_	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	_	_
 17	bomba	bomba	NOUN	_	Gender=Fem|Number=Sing	21	nsubj:pass	_	_
@@ -142942,7 +142942,7 @@
 32	trabalho	trabalho	NOUN	_	Gender=Masc|Number=Sing	30	nmod	_	_
 33	interministerial	interministerial	ADJ	_	Gender=Masc|Number=Sing	32	amod	_	_
 34	,	,	PUNCT	_	_	35	punct	_	_
-35	encarregado	encarregado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	30	acl	_	_
+35	encarregado	encarregar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	30	acl	_	_
 36	de	de	SCONJ	_	_	37	mark	_	_
 37	apresentar	apresentar	VERB	_	VerbForm=Inf	35	xcomp	_	_
 38	estudos	estudo	NOUN	_	Gender=Masc|Number=Plur	37	obj	_	_
@@ -143957,13 +143957,13 @@
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	_	_
 25	utilização	utilização	NOUN	_	Gender=Fem|Number=Sing	22	nmod	_	_
 26	,	,	PUNCT	_	_	27	punct	_	_
-27	breve	breve	NOUN	_	Gender=Masc|Number=Sing	25	conj	_	_
+27	breve	breve	NOUN	_	Gender=Fem|Number=Sing	25	amod	_	_
 28	ou	ou	CCONJ	_	_	29	cc	_	_
-29	prolongada	prolongar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	25	conj	_	_
+29	prolongada	prolongado	ADJ	_	Gender=Fem|Number=Sing	27	conj	_	_
 30	,	,	PUNCT	_	_	36	punct	_	_
 31	de	de	ADP	_	_	33	case	_	_
 32	maiores	grande	ADJ	_	Gender=Fem|Number=Plur	33	amod	_	_
-33	porcentagens	porcentagem	NOUN	_	Gender=Fem|Number=Plur	22	conj	_	_
+33	porcentagens	porcentagem	NOUN	_	Gender=Fem|Number=Plur	25	nmod	_	_
 34	de	de	ADP	_	_	35	case	_	_
 35	biodiesel	biodiesel	NOUN	_	Gender=Masc|Number=Sing	33	nmod	_	_
 36	misturado	misturar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	35	acl	_	_
@@ -144996,7 +144996,7 @@
 3	aparato	aparato	NOUN	_	Gender=Masc|Number=Sing	6	nsubj:pass	_	_
 4	utilizado	utilizar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	3	acl	_	_
 5	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	6	aux:pass	_	_
-6	disposto	disposto	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
+6	disposto	dispor	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
 7	como	como	SCONJ	_	_	8	mark	_	_
 8	mostrado	mostrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	6	advcl	_	_
 9-10	na	_	_	_	_	_	_	_	_
@@ -151251,18 +151251,18 @@
 # text = O óleo de fritura, quando não mais necessário, ou é descartado na rede de esgoto ou enviado para aterros.
 # sent_id = 271-20150810-MONOGRAFIA_0-70
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	_	_
-2	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	0	root	_	_
+2	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	13	nsubj:pass	_	_
 3	de	de	ADP	_	_	4	case	_	_
 4	fritura	fritura	NOUN	_	Gender=Fem|Number=Sing	2	nmod	_	_
 5	,	,	PUNCT	_	_	9	punct	_	_
-6	quando	quando	ADV	_	_	9	advmod	_	_
+6	quando	quando	SCONJ	_	_	9	mark	_	_
 7	não	não	ADV	_	Polarity=Neg	8	advmod	_	_
 8	mais	mais	ADV	_	_	9	advmod	_	_
-9	necessário	necessário	ADJ	_	Gender=Masc|Number=Sing	2	amod	_	_
+9	necessário	necessário	ADJ	_	Gender=Masc|Number=Sing	13	advcl	_	_
 10	,	,	PUNCT	_	_	13	punct	_	_
 11	ou	ou	CCONJ	_	_	13	cc	_	_
 12	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	13	aux:pass	_	_
-13	descartado	descartar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	9	conj	_	_
+13	descartado	descartar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
 14-15	na	_	_	_	_	_	_	_	_
 14	em	em	ADP	_	_	16	case	_	_
 15	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	16	det	_	_
@@ -151270,10 +151270,10 @@
 17	de	de	ADP	_	_	18	case	_	_
 18	esgoto	esgoto	NOUN	_	Gender=Masc|Number=Sing	16	nmod	_	_
 19	ou	ou	CCONJ	_	_	20	cc	_	_
-20	enviado	enviar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	18	conj	_	_
+20	enviado	enviar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	13	conj	_	_
 21	para	para	ADP	_	_	22	case	_	_
 22	aterros	aterro	NOUN	_	Gender=Masc|Number=Plur	20	obl	_	_
-23	.	.	PUNCT	_	_	2	punct	_	_
+23	.	.	PUNCT	_	_	13	punct	_	_
 
 # text = Em alguns casos, é doado ou vendido para a produção de sabão.
 # sent_id = 271-20150810-MONOGRAFIA_0-71
@@ -156384,7 +156384,7 @@
 7	com	com	ADP	_	_	8	case	_	_
 8	metanol	metanol	NOUN	_	Gender=Masc|Number=Sing	6	nmod	_	_
 9	e	e	CCONJ	_	_	10	cc	_	_
-10	mantido	manter	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	8	conj	_	_
+10	mantido	manter	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	4	conj	_	_
 11	sob	sob	ADP	_	_	12	case	_	_
 12	condições	condição	NOUN	_	Gender=Fem|Number=Plur	10	obl	_	_
 13	de	de	ADP	_	_	14	case	_	_
@@ -157643,7 +157643,7 @@
 13-14	tornando-o	_	_	_	_	_	_	_	_
 13	tornando	tornar	VERB	_	VerbForm=Ger	6	advcl	_	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	_	_
-15	avermelhado	avermelhado	NOUN	_	Gender=Masc|Number=Sing	13	obj	_	_
+15	avermelhado	avermelhado	ADJ	_	Gender=Masc|Number=Sing	13	obj	_	_
 16	e	e	CCONJ	_	_	17	cc	_	_
 17	aglomerado	aglomerar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	15	conj	_	_
 18	.	.	PUNCT	_	_	1	punct	_	_
@@ -161357,11 +161357,11 @@
 8	retirado	retirar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
 9	por	por	ADP	_	_	10	case	_	_
 10	filtração	filtração	NOUN	_	Gender=Fem|Number=Sing	8	obl:agent	_	_
-11	e	e	CCONJ	_	_	13	cc	_	_
+11	e	e	CCONJ	_	_	15	cc	_	_
 12	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	13	det	_	_
-13	reação	reação	NOUN	_	Gender=Fem|Number=Sing	10	conj	_	_
+13	reação	reação	NOUN	_	Gender=Fem|Number=Sing	15	nsubj	_	_
 14	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	15	aux:pass	_	_
-15	conduzida	conduzido	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	8	conj	_	_
+15	conduzida	conduzir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	8	conj	_	_
 16	até	até	SCONJ	_	_	17	mark	_	_
 17	atingir	atingir	VERB	_	VerbForm=Inf	15	advcl	_	_
 18	180	180	NUM	_	NumType=Card	19	nummod	_	_
@@ -162665,7 +162665,7 @@
 9	de	de	ADP	_	_	10	case	_	_
 10	fritura	fritura	NOUN	_	Gender=Fem|Number=Sing	8	nmod	_	_
 11	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	12	aux:pass	_	_
-12	estimado	estimado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
+12	estimado	estimar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
 13	,	,	PUNCT	_	_	12	punct	_	_
 14	pois	pois	SCONJ	_	_	16	mark	_	_
 15	serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	16	aux:pass	_	_
@@ -167045,7 +167045,7 @@
 # sent_id = 391-20160914-MONOGRAFIA_0-48
 1	Estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	2	det	_	_
 2	aspectos	aspecto	NOUN	_	Gender=Masc|Number=Plur	4	nsubj	_	_
-3	combinados	combinado	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	2	acl	_	_
+3	combinados	combinar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	2	acl	_	_
 4	aumentam	aumentar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	_	_
 5	consideravelmente	consideravelmente	ADV	_	_	4	advmod	_	_
 6	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	_	_
@@ -168692,7 +168692,7 @@
 
 # text = Associado as extrações de petróleo, obtém-se um subproduto indesejável conhecido como agua produzida, que tem esse nome em referência a produção de petróleo.
 # sent_id = 391-20160914-MONOGRAFIA_0-98
-1	Associado	associado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	7	advcl	_	_
+1	Associado	associar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	7	advcl	_	_
 2	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	3	det	_	_
 3	extrações	extração	NOUN	_	Gender=Fem|Number=Plur	1	obj	_	_
 4	de	de	ADP	_	_	5	case	_	_
@@ -178056,7 +178056,7 @@
 6	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	7	det	_	_
 7	bomba	bomba	NOUN	_	Gender=Fem|Number=Sing	4	obl:agent	_	_
 8	mecânica	mecânico	ADJ	_	Gender=Fem|Number=Sing	7	amod	_	_
-9	conectada	conectado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	7	acl	_	_
+9	conectada	conectar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	7	acl	_	_
 10-11	pela	_	_	_	_	_	_	_	_
 10	por	por	ADP	_	_	12	case	_	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	_	_
@@ -187248,7 +187248,7 @@
 28	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	31	cop	_	_
 29	de	de	ADP	_	_	31	case	_	_
 30	baixo	baixo	ADJ	_	Gender=Masc|Number=Sing	31	amod	_	_
-31	custo	custo	NOUN	_	Gender=Masc|Number=Sing	3	csubj	_	_
+31	custo	custo	NOUN	_	Gender=Masc|Number=Sing	5	conj	_	_
 32	para	para	ADP	_	_	33	case	_	_
 33	indústria	indústria	NOUN	_	Gender=Fem|Number=Sing	31	nmod	_	_
 34-35	do	_	_	_	_	_	_	_	_
@@ -187897,7 +187897,7 @@
 1	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	2	det	_	_
 2	feições	feição	NOUN	_	Gender=Fem|Number=Plur	5	nsubj	_	_
 3	magnéticas	magnético	ADJ	_	Gender=Fem|Number=Plur	2	amod	_	_
-4	alvo	alvo	ADJ	_	Gender=Masc|Number=Sing	2	amod	_	_
+4	alvo	alvo	NOUN	_	Gender=Masc|Number=Sing	2	nmod	_	_
 5	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	_	_
 6	ser	ser	AUX	_	VerbForm=Inf	7	aux:pass	_	_
 7	divididas	dividir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	5	xcomp	_	_
@@ -187907,11 +187907,11 @@
 11	origem	origem	NOUN	_	Gender=Fem|Number=Sing	7	obl	_	_
 12	,	,	PUNCT	_	_	7	punct	_	_
 13	em	em	ADP	_	_	14	case	_	_
-14	estruturais	estrutural	NOUN	_	Gender=Masc|Number=Plur	7	obl	_	_
+14	estruturais	estrutural	ADJ	_	Gender=Masc|Number=Plur	7	obl:arg	_	_
 15	e	e	CCONJ	_	_	16	cc	_	_
-16	associadas	associar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	14	conj	_	_
+16	associadas	associado	ADJ	_	Gender=Fem|Number=Plur	14	conj	_	_
 17	a	a	ADP	_	_	18	case	_	_
-18	corpos	corpo	NOUN	_	Gender=Masc|Number=Plur	16	obl:arg	_	_
+18	corpos	corpo	NOUN	_	Gender=Masc|Number=Plur	16	obl	_	_
 19	magmáticos	magmático	ADJ	_	Gender=Masc|Number=Plur	18	amod	_	_
 20	.	.	PUNCT	_	_	5	punct	_	_
 
@@ -190473,16 +190473,16 @@
 24	distribuídos	distribuir	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	23	acl	_	_
 25	regionalmente	regionalmente	ADV	_	_	24	advmod	_	_
 26	(	(	PUNCT	_	_	28	punct	_	_
-27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	_	_
-28	SW	SW	PROPN	_	Gender=Fem|Number=Sing	23	appos	_	_
+27	a	a	ADP	_	_	28	case	_	_
+28	SW	SW	PROPN	_	Gender=Fem|Number=Sing	24	obl	_	_
 29	,	,	PUNCT	_	_	32	punct	_	_
 30-31	no	_	_	_	_	_	_	_	_
 30	em	em	ADP	_	_	32	case	_	_
 31	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	32	det	_	_
-32	centro	centro	NOUN	_	Gender=Masc|Number=Sing	28	nmod	_	_
+32	centro	centro	NOUN	_	Gender=Masc|Number=Sing	28	conj	_	_
 33	e	e	CCONJ	_	_	35	cc	_	_
-34	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	35	det	_	_
-35	NE	NE	PROPN	_	Gender=Fem|Number=Sing	32	conj	_	_
+34	a	a	ADP	_	_	35	case	_	_
+35	NE	NE	PROPN	_	Gender=Fem|Number=Sing	28	conj	_	_
 36-37	da	_	_	_	_	_	_	_	_
 36	de	de	ADP	_	_	38	case	_	_
 37	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	38	det	_	_
@@ -190490,7 +190490,7 @@
 39	)	)	PUNCT	_	_	28	punct	_	_
 40	e	e	CCONJ	_	_	43	cc	_	_
 41	em	em	ADP	_	_	43	case	_	_
-42	sofisticada	sofisticado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	43	acl	_	_
+42	sofisticada	sofisticado	ADJ	_	Gender=Fem|Number=Sing	43	amod	_	_
 43	técnica	técnica	NOUN	_	Gender=Fem|Number=Sing	17	conj	_	_
 44	de	de	ADP	_	_	45	case	_	_
 45	reintegração	reintegração	NOUN	_	Gender=Fem|Number=Sing	43	nmod	_	_
@@ -190600,7 +190600,7 @@
 23	)	)	PUNCT	_	_	7	punct	_	_
 24	teria	ter	AUX	_	Mood=Cnd|Number=Sing|Person=3|VerbForm=Fin	26	aux	_	_
 25	sido	ser	AUX	_	VerbForm=Part	26	aux:pass	_	_
-26	condicionada	condicionado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
+26	condicionada	condicionar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
 27	por	por	ADP	_	_	28	case	_	_
 28	reativações	reativação	NOUN	_	Gender=Fem|Number=Plur	26	obl:agent	_	_
 29	de	de	ADP	_	_	30	case	_	_
@@ -194098,12 +194098,12 @@
 21	escarpados	escarpar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	20	acl	_	_
 22	,	,	PUNCT	_	_	24	punct	_	_
 23	muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	24	det	_	_
-24	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	4	obl	_	_
+24	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	27	nmod	_	_
 25	de	de	ADP	_	_	27	case	_	_
 26	difícil	difícil	ADJ	_	Gender=Masc|Number=Sing	27	amod	_	_
-27	acesso	acesso	NOUN	_	Gender=Masc|Number=Sing	24	nmod	_	_
+27	acesso	acesso	NOUN	_	Gender=Masc|Number=Sing	21	conj	_	_
 28	e	e	CCONJ	_	_	29	cc	_	_
-29	recobertos	recobrir	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	24	conj	_	_
+29	recobertos	recobrir	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	21	conj	_	_
 30	por	por	ADP	_	_	32	case	_	_
 31	densa	denso	ADJ	_	Gender=Fem|Number=Sing	32	amod	_	_
 32	vegetação	vegetação	NOUN	_	Gender=Fem|Number=Sing	29	obl:agent	_	_
@@ -197264,7 +197264,7 @@
 1	Secundariamente	secundariamente	ADV	_	_	4	advmod	_	_
 2	,	,	PUNCT	_	_	1	punct	_	_
 3	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	4	aux:pass	_	_
-4	empregados	empregado	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	_	_
+4	empregados	empregar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	_	_
 5	dados	dado	NOUN	_	Gender=Masc|Number=Plur	4	nsubj:pass	_	_
 6	de	de	ADP	_	_	7	case	_	_
 7	sísmica	sísmica	NOUN	_	Gender=Fem|Number=Sing	5	nmod	_	_
@@ -199731,7 +199731,7 @@
 17	a	ela	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	16	obj	_	_
 18	de	de	ADP	_	_	19	case	_	_
 19	forma	forma	NOUN	_	Gender=Fem|Number=Sing	17	nmod	_	_
-20	indiscriminada	indiscriminado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	19	acl	_	_
+20	indiscriminada	indiscriminado	ADJ	_	Gender=Fem|Number=Sing	19	amod	_	_
 21	,	,	PUNCT	_	_	22	punct	_	_
 22	extrapolando	extrapolar	VERB	_	VerbForm=Ger	16	advcl	_	_
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	_	_
@@ -199746,7 +199746,7 @@
 32	,	,	PUNCT	_	_	22	punct	_	_
 33-34	no	_	_	_	_	_	_	_	_
 33	em	em	ADP	_	_	35	case	_	_
-34	o	o	DET	_	Gender=Masc|Number=Sing|PronType=Art	35	det	_	_
+34	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	35	det	_	_
 35	qual	qual	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	39	obl	_	_
 36	nem	nem	ADV	_	_	37	advmod	_	_
 37	sempre	sempre	ADV	_	_	39	advmod	_	_
@@ -200023,7 +200023,7 @@
 17	,	,	PUNCT	_	_	2	punct	_	_
 18	também	também	ADV	_	_	20	advmod	_	_
 19	largamente	largamente	ADV	_	_	20	advmod	_	_
-20	empregado	empregado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	11	acl	_	_
+20	empregado	empregar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	11	acl	_	_
 21	como	como	ADP	_	_	22	case	_	_
 22	ferramenta	ferramenta	NOUN	_	Gender=Fem|Number=Sing	20	xcomp	_	_
 23	de	de	ADP	_	_	24	case	_	_
@@ -200067,7 +200067,7 @@
 58	seja	ser	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	57	fixed	_	_
 59	,	,	PUNCT	_	_	61	punct	_	_
 60	em	em	ADP	_	_	61	case	_	_
-61	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	51	conj	_	_
+61	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	54	conj	_	_
 62	verticalmente	verticalmente	ADV	_	_	63	advmod	_	_
 63	magnetizado	magnetizado	ADJ	_	Gender=Masc|Number=Sing	61	amod	_	_
 64	.	.	PUNCT	_	_	26	punct	_	_
@@ -200559,7 +200559,7 @@
 45	mais	mais	ADV	_	_	48	advmod	_	_
 46	ou	ou	CCONJ	_	_	48	cc	_	_
 47	menos	menos	ADV	_	_	48	advmod	_	_
-48	indiscriminada	indiscriminado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	44	acl	_	_
+48	indiscriminada	indiscriminado	ADJ	_	Gender=Fem|Number=Sing	44	amod	_	_
 49	.	.	PUNCT	_	_	2	punct	_	_
 
 # text = Se for possível assumir, entretanto, com bom nível de confiabilidade, que o ambiente geológico estudado apresente contatos verticais regionalizados entre as unidades litológicas, tanto mais consistente será a interpretação dos limites de corpos magnetizados, independentemente de suas direções de magnetização.
@@ -203036,11 +203036,11 @@
 12	fonte	fonte	NOUN	_	Gender=Fem|Number=Sing	10	obl	_	_
 13	única	único	ADJ	_	Gender=Fem|Number=Sing	12	amod	_	_
 14	,	,	PUNCT	_	_	19	punct	_	_
-15	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	19	cc	_	_
+15	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	19	cc	_	MWEPOS=CCONJ
 16	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	fixed	_	_
 17	,	,	PUNCT	_	_	15	punct	_	_
 18	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	19	expl	_	_
-19	relaciona	relacionar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	12	conj	_	_
+19	relaciona	relacionar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	10	conj	_	_
 20	a	a	ADP	_	_	23	case	_	_
 21	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	23	det	_	_
 22	único	único	ADJ	_	Gender=Masc|Number=Sing	23	amod	_	_
@@ -203891,7 +203891,7 @@
 12	índice	índice	NOUN	_	Gender=Masc|Number=Sing	15	nsubj:pass	_	_
 13	estrutural	estrutural	ADJ	_	Gender=Masc|Number=Sing	12	amod	_	_
 14	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	aux:pass	_	_
-15	estimado	estimado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	8	acl:relcl	_	_
+15	estimado	estimar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	8	acl:relcl	_	_
 16	como	como	ADP	_	_	19	case	_	_
 17	mais	mais	ADV	_	_	19	advmod	_	_
 18	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	19	det	_	_
@@ -205928,7 +205928,7 @@
 63	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	64	det	_	_
 64	vetor	vetor	NOUN	_	Gender=Masc|Number=Sing	60	nmod	_	_
 65	p	p	NOUN	_	Gender=Masc|Number=Sing	64	nmod	_	_
-66	estimado	estimado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	64	acl	_	_
+66	estimado	estimar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	64	acl	_	_
 67	.	.	PUNCT	_	_	14	punct	_	_
 
 # text = A estimativa de p é então atualizada a cada iteração K na forma (15) (16)
@@ -206267,8 +206267,8 @@
 6	quando	quando	SCONJ	_	_	14	mark	_	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	_	_
 8	parâmetro	parâmetro	NOUN	_	Gender=Masc|Number=Sing	14	nsubj	_	_
-9	estimado	estimado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	8	acl	_	_
-10	pj	pj	PROPN	_	Gender=Masc|Number=Sing	9	xcomp	_	_
+9	estimado	estimar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	8	acl	_	_
+10	pj	pj	PROPN	_	Gender=Masc|Number=Sing	8	nmod	_	_
 11	(	(	PUNCT	_	_	12	punct	_	_
 12	K	K	PROPN	_	Gender=Masc|Number=Sing	8	appos	_	_
 13	)	)	PUNCT	_	_	12	punct	_	_
@@ -210352,7 +210352,7 @@
 32	seção	seção	NOUN	_	Gender=Fem|Number=Sing	30	nmod	_	_
 33	constante	constante	ADJ	_	Gender=Fem|Number=Sing	32	amod	_	_
 34	e	e	CCONJ	_	_	35	cc	_	_
-35	coordenadas	coordenada	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	33	conj	_	_
+35	coordenadas	coordenada	NOUN	_	Gender=Fem|Number=Plur	32	conj	_	_
 36	fixas	fixo	ADJ	_	Gender=Fem|Number=Plur	35	amod	_	_
 37	através	através	ADV	_	_	40	case	_	MWEPOS=ADP
 38-39	do	_	_	_	_	_	_	_	_
@@ -212099,7 +212099,7 @@
 7	,	,	PUNCT	_	_	5	punct	_	_
 8	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	_	_
 9	ser	ser	AUX	_	VerbForm=Inf	10	aux:pass	_	_
-10	entendidos	entendidos	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	8	xcomp	_	_
+10	entendidos	entender	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	8	xcomp	_	_
 11	como	como	ADP	_	_	12	case	_	_
 12	áreas	área	NOUN	_	Gender=Fem|Number=Plur	10	xcomp	_	_
 13	,	,	PUNCT	_	_	14	punct	_	_
@@ -212564,7 +212564,7 @@
 22	com	com	ADP	_	_	25	mark	_	MWEPOS=SCONJ
 23	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	22	fixed	_	_
 24	a	a	ADP	_	_	22	fixed	_	_
-25	complementar	complementar	NOUN	_	Gender=Fem|Number=Sing	13	advcl	_	_
+25	complementar	complementar	VERB	_	VerbForm=Inf	13	advcl	_	_
 26	e	e	CCONJ	_	_	27	cc	_	_
 27	detalhar	detalhar	VERB	_	VerbForm=Inf	23	conj	_	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	_	_
@@ -213488,7 +213488,7 @@
 # sent_id = 20-20140904-TESEDSC_0-732
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	_	_
 2	calha	calha	NOUN	_	Gender=Fem|Number=Sing	12	nsubj	_	_
-3	condicionada	condicionado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	2	acl	_	_
+3	condicionada	condicionar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	2	acl	_	_
 4-5	pela	_	_	_	_	_	_	_	_
 4	por	por	ADP	_	_	6	case	_	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	_	_
@@ -216850,7 +216850,7 @@
 32	de	de	ADP	_	_	30	fixed	_	_
 33	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	34	det	_	_
 34	nível	nível	NOUN	_	Gender=Masc|Number=Sing	23	obl	_	_
-35	estimado	estimado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	34	acl	_	_
+35	estimado	estimar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	34	acl	_	_
 36	de	de	ADP	_	_	38	case	_	_
 37	1.100	1.100	NUM	_	NumType=Card	38	nummod	_	_
 38	metros	metro	NOUN	_	Gender=Masc|Number=Plur	34	nmod	_	_
@@ -229531,7 +229531,7 @@
 2	depocentro	depocentro	NOUN	_	Gender=Masc|Number=Sing	9	nsubj	_	_
 3	,	,	PUNCT	_	_	4	punct	_	_
 4	por	por	ADP	_	_	9	obl	_	MWEPOS=ADV
-5	sua	sua	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Prs	4	fixed	_	_
+5	sua	sua	DET	_	Gender=Fem|Number=Sing|PronType=Prs	4	fixed	_	_
 6	vez	vez	NOUN	_	Gender=Masc|Number=Sing	4	fixed	_	_
 7	,	,	PUNCT	_	_	4	punct	_	_
 8	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	9	expl	_	_
@@ -229542,7 +229542,7 @@
 13	de	de	ADP	_	_	15	case	_	_
 14	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	15	det	_	_
 15	calha	calha	NOUN	_	Gender=Fem|Number=Sing	12	nmod	_	_
-16	condicionada	condicionado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	15	acl	_	_
+16	condicionada	condicionar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	15	acl	_	_
 17	,	,	PUNCT	_	_	18	punct	_	_
 18	paralela	paralelo	ADJ	_	Gender=Fem|Number=Sing	15	amod	_	_
 19-20	ao	_	_	_	_	_	_	_	_
@@ -229880,7 +229880,7 @@
 14	pacote	pacote	NOUN	_	Gender=Masc|Number=Sing	27	nsubj	_	_
 15	sedimentar	sedimentar	ADJ	_	Gender=Masc|Number=Sing	14	amod	_	_
 16	(	(	PUNCT	_	_	17	punct	_	_
-17	expresso	expresso	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	14	acl	_	_
+17	expresso	expressar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	14	acl	_	_
 18-19	pelos	_	_	_	_	_	_	_	_
 18	por	por	ADP	_	_	20	case	_	_
 19	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	20	det	_	_
@@ -232873,7 +232873,7 @@
 71	nível	nível	NOUN	_	Gender=Masc|Number=Sing	67	conj	_	_
 72	de	de	ADP	_	_	73	case	_	_
 73	base	base	NOUN	_	Gender=Fem|Number=Sing	71	nmod	_	_
-74	estimado	estimado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	71	acl	_	_
+74	estimado	estimar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	71	acl	_	_
 75	.	.	PUNCT	_	_	16	punct	_	_
 
 # text = O emprego destes dois critérios, utilizados simultaneamente para fins comparativos, demonstrou que nos casos onde as anomalias provêm de fonte única, ou são fracamente influenciadas por fontes adjacentes, ambos concordam fornecendo as mesmas estimativas e os mesmos índices estruturais.
@@ -237486,7 +237486,7 @@
 20	e	e	CCONJ	_	_	22	cc	_	_
 21	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	22	det	_	_
 22	calha	calha	NOUN	_	Gender=Fem|Number=Sing	14	conj	_	_
-23	condicionada	condicionado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	22	acl	_	_
+23	condicionada	condicionar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	22	acl	_	_
 24	,	,	PUNCT	_	_	23	punct	_	_
 25	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	26	cop	_	_
 26	deslocadas	deslocado	ADJ	_	Gender=Fem|Number=Plur	11	ccomp	_	_
@@ -238543,7 +238543,7 @@
 36	de	de	ADP	_	_	37	case	_	_
 37	sólidos	sólido	NOUN	_	Gender=Masc|Number=Plur	35	nmod	_	_
 38	e	e	CCONJ	_	_	39	cc	_	_
-39	filtrado	filtrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	37	conj	_	_
+39	filtrado	filtrado	NOUN	_	Gender=Masc|Number=Sing	37	conj	_	_
 40	.	.	PUNCT	_	_	13	punct	_	_
 
 # text = Para isto, é necessário que se tenha controle tanto sobre a viscosidade quanto sobre a densidade do fluido.
@@ -241092,18 +241092,18 @@
 5	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	6	det	_	_
 6	estado	estado	NOUN	_	Gender=Masc|Number=Sing	4	obj	_	_
 7	semirrígido	semirrígir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	6	acl	_	_
-8	quando	quando	ADV	_	_	11	advmod	_	_
+8	quando	quando	SCONJ	_	_	11	mark	_	_
 9	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	11	cop	_	_
 10	em	em	ADP	_	_	11	case	_	_
 11	repouso	repouso	NOUN	_	Gender=Masc|Number=Sing	4	advcl	_	_
 12	e	e	CCONJ	_	_	13	cc	_	_
-13	voltam	voltar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	11	conj	_	_
+13	voltam	voltar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	conj	_	_
 14	a	a	SCONJ	_	_	15	mark	_	_
 15	adquirir	adquirir	VERB	_	VerbForm=Inf	13	xcomp	_	_
 16	estado	estado	NOUN	_	Gender=Masc|Number=Sing	15	obj	_	_
 17	de	de	ADP	_	_	18	case	_	_
 18	fluidez	fluidez	NOUN	_	Gender=Fem|Number=Sing	16	nmod	_	_
-19	quando	quando	ADV	_	_	23	advmod	_	_
+19	quando	quando	SCONJ	_	_	23	mark	_	_
 20	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	23	cop	_	_
 21	novamente	novamente	ADV	_	_	23	advmod	_	_
 22	em	em	ADP	_	_	23	case	_	_
@@ -242629,7 +242629,7 @@
 6	de	de	ADP	_	_	7	case	_	_
 7	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	5	nmod	_	_
 8	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	9	aux:pass	_	_
-9	definidas	definido	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	_	_
+9	definidas	definir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	_	_
 10	a	a	ADP	_	_	13	mark	_	_
 11	fim	fim	NOUN	_	Gender=Masc|Number=Sing	10	fixed	_	_
 12	de	de	ADP	_	_	10	fixed	_	_
@@ -242871,7 +242871,7 @@
 36	sólidos	sólido	NOUN	_	Gender=Masc|Number=Plur	34	nmod	_	_
 37	e	e	CCONJ	_	_	39	cc	_	_
 38	de	de	ADP	_	_	39	case	_	_
-39	filtrado	filtrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	36	conj	_	_
+39	filtrado	filtrado	NOUN	_	Gender=Masc|Number=Sing	36	conj	_	_
 40	,	,	PUNCT	_	_	41	punct	_	_
 41	minimizando	minimizar	VERB	_	VerbForm=Ger	32	advcl	_	_
 42	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	43	det	_	_
@@ -245941,7 +245941,7 @@
 10	reológicos	reológico	ADJ	_	Gender=Masc|Number=Plur	9	amod	_	_
 11	,	,	PUNCT	_	_	12	punct	_	_
 12	por	por	ADP	_	_	16	obl	_	MWEPOS=ADV
-13	sua	sua	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Prs	12	fixed	_	_
+13	sua	sua	DET	_	Gender=Fem|Number=Sing|PronType=Prs	12	fixed	_	_
 14	vez	vez	NOUN	_	Gender=Masc|Number=Sing	12	fixed	_	_
 15	,	,	PUNCT	_	_	12	punct	_	_
 16	permite	permitir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	_	_
@@ -245975,7 +245975,7 @@
 43	,	,	PUNCT	_	_	46	punct	_	_
 44	além	além	ADV	_	_	46	cc	_	MWEPOS=CCONJ
 45	de	de	ADP	_	_	44	fixed	_	_
-46	especificar	especificar	VERB	_	VerbForm=Inf	35	conj	_	_
+46	especificar	especificar	VERB	_	VerbForm=Inf	19	conj	_	_
 47	e	e	CCONJ	_	_	48	cc	_	_
 48	qualificar	qualificar	VERB	_	VerbForm=Inf	46	conj	_	_
 49	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	48	obj	_	_
@@ -255627,7 +255627,7 @@
 3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	_	_
 4	sistema	sistema	NOUN	_	Gender=Masc|Number=Sing	1	obj	_	_
 5	como	como	ADP	_	_	6	case	_	_
-6	incompressível	incompressível	NOUN	_	Gender=Masc|Number=Sing	4	nmod	_	_
+6	incompressível	incompressível	ADJ	_	Gender=Masc|Number=Sing	2	xcomp	_	_
 7	,	,	PUNCT	_	_	1	punct	_	_
 8	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	9	det	_	_
 9	volumes	volume	NOUN	_	Gender=Masc|Number=Plur	12	nsubj	_	_
@@ -255655,9 +255655,9 @@
 29	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	27	nmod	_	_
 30	invasor	invasor	ADJ	_	Gender=Masc|Number=Sing	29	amod	_	_
 31	ou	ou	CCONJ	_	_	32	cc	_	_
-32	fluido	fluido	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	30	conj	_	_
+32	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	29	conj	_	_
 33	de	de	ADP	_	_	34	case	_	_
-34	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	29	nmod	_	_
+34	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	32	nmod	_	_
 35	perdido	perder	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	32	acl	_	_
 36	para	para	ADP	_	_	38	case	_	_
 37	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	38	det	_	_
@@ -258592,7 +258592,7 @@
 26	,	,	PUNCT	_	_	29	punct	_	_
 27	tendo	ter	AUX	_	VerbForm=Ger	29	aux	_	_
 28	sido	ser	AUX	_	VerbForm=Part	29	aux:pass	_	_
-29	suposto	suposto	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	20	advcl	_	_
+29	suposto	supor	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	20	advcl	_	_
 30	que	que	SCONJ	_	_	35	mark	_	_
 31	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	32	det	_	_
 32	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	35	nsubj:pass	_	_
@@ -263012,8 +263012,8 @@
 5	,	,	PUNCT	_	_	4	punct	_	_
 6	pois	pois	SCONJ	_	_	10	mark	_	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	_	_
-8	KH>LMNOPãQ	KH>LMNOPãQ	PROPN	_	Gender=Masc|Number=Sing	10	nsubj	_	_
-9	fluido	fluido	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	8	acl	_	_
+8	KH>LMNOPãQ	KH>LMNOPãQ	PROPN	_	Gender=Masc|Number=Sing	10	nsubj	_	tokenização
+9	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	10	nsubj	_	_
 10	transporta	transportar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	advcl	_	_
 11	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	13	det	_	_
 12	maior	grande	ADJ	_	Gender=Fem|Number=Sing	13	amod	_	_
@@ -269027,26 +269027,26 @@
 8	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	9	cop	_	_
 9	tixotrópicos	tixotrópico	ADJ	_	Gender=Masc|Number=Plur	0	root	_	_
 10	,	,	PUNCT	_	_	11	punct	_	_
-11	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	14	cc	_	_
+11	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	14	cc	_	MWEPOS=CCONJ
 12	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	fixed	_	_
 13	,	,	PUNCT	_	_	11	punct	_	_
 14	adquirem	adquirir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	9	conj	_	_
 15	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	16	det	_	_
 16	estado	estado	NOUN	_	Gender=Masc|Number=Sing	14	obj	_	_
 17	semi-rígido	semi-rígido	ADJ	_	Gender=Masc|Number=Sing	16	amod	_	_
-18	quando	quando	ADV	_	_	21	advmod	_	_
+18	quando	quando	SCONJ	_	_	21	mark	_	_
 19	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	21	cop	_	_
 20	em	em	ADP	_	_	21	case	_	_
 21	repouso	repouso	NOUN	_	Gender=Masc|Number=Sing	14	advcl	_	_
 22	e	e	CCONJ	_	_	23	cc	_	_
-23	voltam	voltar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	21	conj	_	_
+23	voltam	voltar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	14	conj	_	_
 24	a	a	SCONJ	_	_	25	mark	_	_
 25	adquirir	adquirir	VERB	_	VerbForm=Inf	23	xcomp	_	_
 26	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	27	det	_	_
 27	estado	estado	NOUN	_	Gender=Masc|Number=Sing	25	obj	_	_
 28	de	de	ADP	_	_	29	case	_	_
 29	fluidez	fluidez	NOUN	_	Gender=Fem|Number=Sing	27	nmod	_	_
-30	quando	quando	ADV	_	_	34	advmod	_	_
+30	quando	quando	SCONJ	_	_	34	mark	_	_
 31	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	34	cop	_	_
 32	novamente	novamente	ADV	_	_	34	advmod	_	_
 33	em	em	ADP	_	_	34	case	_	_
@@ -271907,7 +271907,7 @@
 9	,	,	PUNCT	_	_	10	punct	_	_
 10	transparentes	transparente	ADJ	_	Gender=Masc|Number=Plur	8	conj	_	_
 11	ou	ou	CCONJ	_	_	12	cc	_	_
-12	translúcidos	translúcido	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	8	conj	_	_
+12	translúcidos	translúcido	ADJ	_	Gender=Masc|Number=Plur	10	conj	_	_
 13	,	,	PUNCT	_	_	14	punct	_	_
 14	monofásicos	monofásico	ADJ	_	Gender=Masc|Number=Plur	8	conj	_	_
 15	,	,	PUNCT	_	_	16	punct	_	_
@@ -271933,7 +271933,7 @@
 34	de	de	ADP	_	_	35	case	_	_
 35	tensoativo	tensoativo	NOUN	_	Gender=Masc|Number=Sing	33	nmod	_	_
 36	e	e	CCONJ	_	_	37	cc	_	_
-37	cotensoativo	cotensoativo	ADJ	_	Gender=Masc|Number=Sing	35	conj	_	_
+37	cotensoativo	cotensoativo	NOUN	_	Gender=Masc|Number=Sing	35	conj	_	_
 38	(	(	PUNCT	_	_	39	punct	_	_
 39	ROBB	ROBB	PROPN	_	Gender=Masc|Number=Sing	4	parataxis	_	_
 40	,	,	PUNCT	_	_	39	flat:name	_	_
@@ -278325,7 +278325,7 @@
 58	método	método	NOUN	_	Gender=Masc|Number=Sing	55	nmod	_	_
 59	a	a	SCONJ	_	_	61	mark	_	_
 60	ser	ser	AUX	_	VerbForm=Inf	61	aux:pass	_	_
-61	empregado	empregado	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	58	advcl	_	_
+61	empregado	empregar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	58	advcl	_	_
 62-63	vincula-se	_	_	_	_	_	_	_	_
 62	vincula	vincular	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	52	ccomp	_	_
 63	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	62	expl	_	_
@@ -278353,7 +278353,7 @@
 82	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	79	nmod	_	_
 83-84	no	_	_	_	_	_	_	_	_
 83	em	em	ADP	_	_	85	case	_	_
-84	o	o	DET	_	Gender=Masc|Number=Sing|PronType=Art	85	det	_	_
+84	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	85	det	_	_
 85	qual	qual	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	86	obl	_	_
 86	há	haver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	82	acl:relcl	_	_
 87	necessidade	necessidade	NOUN	_	Gender=Fem|Number=Sing	86	obj	_	_
@@ -279098,7 +279098,7 @@
 8	por	por	ADP	_	_	9	case	_	_
 9	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	7	nmod	_	_
 10	e	e	CCONJ	_	_	11	cc	_	_
-11	derivados	derivar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	9	conj	_	_
+11	derivados	derivado	NOUN	_	Gender=Masc|Number=Plur	9	conj	_	_
 12	,	,	PUNCT	_	_	33	punct	_	_
 13-14	dessa	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	_	_
@@ -279164,7 +279164,7 @@
 8	por	por	ADP	_	_	9	case	_	_
 9	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	7	obl:agent	_	_
 10	e	e	CCONJ	_	_	11	cc	_	_
-11	derivados	derivar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	9	conj	_	_
+11	derivados	derivado	NOUN	_	Gender=Masc|Number=Plur	9	conj	_	_
 12	,	,	PUNCT	_	_	11	punct	_	_
 13	regulamentações	regulamentação	NOUN	_	Gender=Fem|Number=Plur	17	nsubj	_	_
 14	nacionais	nacional	ADJ	_	Gender=Fem|Number=Plur	13	amod	_	_
@@ -283043,7 +283043,7 @@
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	_	_
 9	Ceará	Ceará	PROPN	_	Gender=Masc|Number=Sing	6	nmod	_	_
 10	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	aux:pass	_	_
-11	conduzida	conduzido	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
+11	conduzida	conduzir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	_	_
 12-13	pelo	_	_	_	_	_	_	_	_
 12	por	por	ADP	_	_	14	case	_	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	_	_
@@ -285353,7 +285353,7 @@
 
 # text = As ações das equipes de resposta e a maneira em que as barreiras para contenção e retirada desse óleo derramado irão se instalar no ambiente marinho estão dependentes do grau em que o ambiente se encontra (DEVIDS, 2008).  Amplitudes de maré: Os mares de sizígia são marés que atingem maiores amplitudes conhecidas como marés vivas, e em um derrame podem atingir maiores áreas.
 # sent_id = 411-20150702-MONOGRAFIA_0-204
-1	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	2	det	_	_
+1	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	2	det	_	sentenciação
 2	ações	ação	NOUN	_	Gender=Fem|Number=Plur	31	nsubj	_	_
 3-4	das	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	_	_
@@ -293394,7 +293394,7 @@
 4	e	e	CCONJ	_	_	6	cc	_	_
 5	não	não	ADV	_	Polarity=Neg	6	advmod	_	_
 6	sofrerem	sofrer	VERB	_	Number=Plur|Person=3|VerbForm=Inf	2	conj	_	_
-7	influencia	influencia	NOUN	_	Gender=Fem|Number=Sing	6	obj	_	_
+7	influencia	influência	NOUN	_	Gender=Fem|Number=Sing	6	obj	_	_
 8-9	do	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	_	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	_	_
@@ -298679,7 +298679,7 @@
 6	temos	ter	VERB	_	Mood=Ind|Number=Plur|Person=1|Tense=Pres|VerbForm=Fin	10	advcl	_	_
 7	dois	dois	NUM	_	NumType=Card	8	nummod	_	_
 8	processos	processo	NOUN	_	Gender=Masc|Number=Plur	6	obj	_	_
-9	combinados	combinado	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	8	acl	_	_
+9	combinados	combinar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	8	acl	_	_
 10	podemos	poder	VERB	_	Mood=Ind|Number=Plur|Person=1|Tense=Pres|VerbForm=Fin	3	ccomp	_	_
 11	aumentar	aumentar	VERB	_	VerbForm=Inf	10	xcomp	_	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	_	_
